modules/zip/zip_packer.h

```diff
@@ -52,9 +53,18 @@ class ZIPPacker : public RefCounted {
 		APPEND_ADDINZIP = 2,
 	};
 
+	enum CompressionLevel {
+		COMPRESSION_DEFAULT = 0,
+		COMPRESSION_NONE = 1,
+		COMPRESSION_BEST_SPEED = 2,
+		COMPRESSION_BEST = 3,
+	};
```

Comment:
If values are ordered, the numbers can be removed:

```suggestion
	enum CompressionLevel {
		COMPRESSION_DEFAULT,
		COMPRESSION_NONE,
		COMPRESSION_BEST_SPEED,
		COMPRESSION_BEST,
	};
```